apps/web/pages/event-types/[type].tsx

```diff
@@ -174,6 +175,8 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
     eventType.customInputs.sort((a, b) => a.id - b.id) || []
   );
   const [tokensList, setTokensList] = useState<Array<Token>>([]);
+  const [enableSeats, setEnableSeats] = useState(eventType.seatsPerTimeSlot ? true : false);
```

Comment:
Ternaries for only `? true : false;` are discouraged by a eslint rule
https://eslint.org/docs/rules/no-unneeded-ternary
For this would be `useState(!!eventType.seatsPerTimeSlot);`